<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar:horizontal{
		border: none;
		background-color:rgb(191, 191, 191);
		height: 10px;
		margin: 0 15px 0 15px;
		border-radius: 2px;
}

QScrollBar::handle:horizontal{
	background-color: rgb(199, 199, 199);
	border-radius: 7px;
}

QScrollBar::sub-line:horizontal {
	border: none;
	backround-color: rgba(255, 255, 255, 0);
	height: 15px;
	border-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::add-line:horizontal {
	border: none;
	backround-color: rgba(255, 255, 255, 0);
	height: 15px;
	border-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
/*
QScrollBar::left-arrow:horizontal, QScrollBar::right-arrow:horizontal {
	background :none;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal{
	background: none;
}*/</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>501</width>
     <height>271</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-8</number>
   </property>
   <property name="styleSheet">
    <string notr="true">
border: 1px solid rgba(255, 255, 255, 0.77);

border-radius: 10px;
font-family: Roboto;


</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>fio</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>good</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>bad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>reg</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>840</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: rgba(121, 119, 239, 0.6);
border: 1px solid rgba(255, 255, 255, 0.77);
box-sizing: border-box;
border-radius: 17px;
font-family: Roboto;
font-size: 14px;
}

QPushButton:hover{
	background: rgb(129, 127, 255);
}

QPushButton:pressed{
	background: rgb(108, 106, 213);
	color: rgb(77, 77, 77);
}</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>840</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: rgba(121, 119, 239, 0.6);
border: 1px solid rgba(255, 255, 255, 0.77);
box-sizing: border-box;
border-radius: 17px;
font-family: Roboto;
font-size: 14px;
}

QPushButton:hover{
	background: rgb(129, 127, 255);
}

QPushButton:pressed{
	background: rgb(108, 106, 213);
	color: rgb(77, 77, 77);
}</string>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>840</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: rgba(121, 119, 239, 0.6);
border: 1px solid rgba(255, 255, 255, 0.77);
box-sizing: border-box;
border-radius: 17px;
font-family: Roboto;
font-size: 14px;
}

QPushButton:hover{
	background: rgb(129, 127, 255);
}

QPushButton:pressed{
	background: rgb(108, 106, 213);
	color: rgb(77, 77, 77);
}</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="autoButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>850</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: rgba(121, 119, 239, 0.6);
border: 1px solid rgba(255, 255, 255, 0.77);
box-sizing: border-box;
border-radius: 15px;
font-family: Roboto;
font-size: 14px;
}

QPushButton:hover{
	background: rgb(129, 127, 255);
}

QPushButton:pressed{
	background: rgb(108, 106, 213);
	color: rgb(77, 77, 77);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(237, 237, 237);
border: 1px solid black;
border-radius: 12px;</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: rgba(121, 119, 239, 0.6);
border: 1px solid rgba(255, 255, 255, 0.77);
box-sizing: border-box;
border-radius: 12px;
font-family: Roboto;
font-size: 14px;
}

QPushButton:hover{
	background: rgb(129, 127, 255);
}

QPushButton:pressed{
	background: rgb(108, 106, 213);
	color: rgb(77, 77, 77);
}</string>
    </property>
    <property name="text">
     <string>Подробнее</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(237, 237, 237);
border: 1px solid black;
border-radius: 12px;</string>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: rgba(121, 119, 239, 0.6);
border: 1px solid rgba(255, 255, 255, 0.77);
box-sizing: border-box;
border-radius: 12px;
font-family: Roboto;
font-size: 14px;
}

QPushButton:hover{
	background: rgb(129, 127, 255);
}

QPushButton:pressed{
	background: rgb(108, 106, 213);
	color: rgb(77, 77, 77);
}</string>
    </property>
    <property name="text">
     <string>Подробнее</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(237, 237, 237);
border: 1px solid black;
border-radius: 12px;</string>
   </property>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: rgba(121, 119, 239, 0.6);
border: 1px solid rgba(255, 255, 255, 0.77);
box-sizing: border-box;
border-radius: 12px;
font-family: Roboto;
font-size: 14px;
}

QPushButton:hover{
	background: rgb(129, 127, 255);
}

QPushButton:pressed{
	background: rgb(108, 106, 213);
	color: rgb(77, 77, 77);
}</string>
    </property>
    <property name="text">
     <string>Подробнее</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(237, 237, 237);
border: 1px solid black;
border-radius: 12px;</string>
   </property>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: rgba(121, 119, 239, 0.6);
border: 1px solid rgba(255, 255, 255, 0.77);
box-sizing: border-box;
border-radius: 12px;
font-family: Roboto;
font-size: 14px;
}

QPushButton:hover{
	background: rgb(129, 127, 255);
}

QPushButton:pressed{
	background: rgb(108, 106, 213);
	color: rgb(77, 77, 77);
}</string>
    </property>
    <property name="text">
     <string>Подробнее</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(237, 237, 237);
border: 1px solid black;
border-radius: 12px;</string>
   </property>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: rgba(121, 119, 239, 0.6);
border: 1px solid rgba(255, 255, 255, 0.77);
box-sizing: border-box;
border-radius: 12px;
font-family: Roboto;
font-size: 14px;
}

QPushButton:hover{
	background: rgb(129, 127, 255);
}

QPushButton:pressed{
	background: rgb(108, 106, 213);
	color: rgb(77, 77, 77);
}</string>
    </property>
    <property name="text">
     <string>Подробнее</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(237, 237, 237);
border: 1px solid black;
border-radius: 12px;</string>
   </property>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: rgba(121, 119, 239, 0.6);
border: 1px solid rgba(255, 255, 255, 0.77);
box-sizing: border-box;
border-radius: 12px;
font-family: Roboto;
font-size: 14px;
}

QPushButton:hover{
	background: rgb(129, 127, 255);
}

QPushButton:pressed{
	background: rgb(108, 106, 213);
	color: rgb(77, 77, 77);
}</string>
    </property>
    <property name="text">
     <string>Подробнее</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
